src/customerInfo.ts

```diff
@@ -48,13 +48,25 @@ export interface PurchasesEntitlementInfo {
      *
      * @note: Entitlement may still be active even if user has unsubscribed. Check the `isActive` property.
      */
-    readonly unsubscribeDetectedAt: string | null;
+    readonly unsubscribedDetectedAtMillis: number | null;
+    /**
+     * The date in milliseconds an unsubscribe was detected. Can be `null`.
+     *
+     * @note: Entitlement may still be active even if user has unsubscribed. Check the `isActive` property.
+     */
+    readonly unsubscribedDetectedAt: string | null;
```

Comment:
purchases-hybrid-common sends `unsubscribeDetectedAt` https://github.com/RevenueCat/purchases-hybrid-common/blob/4.14.3/android/src/main/java/com/revenuecat/purchases/hybridcommon/mappers/EntitlementInfoMapper.kt#L20 so this change will break the EntitlementInfo.

I am a bit confused about this change, since we actually recently changed it from sending `unsubscribedDetectedAt` to `unsubscribeDetectedAt` (see https://github.com/RevenueCat/purchases-hybrid-common/pull/345).. Were you getting issues with this field?

Reply:
Ah, it seems then that the data of an entitlement returned to version 4.12.X and the types do not align.

entitlement comes back with `unsubscribedDetectedAt`, but displays a type error as it thinks it should be `unsubscribeDetectedAt`

Reply:
Added a few images to the PR to add color